astroquery/gaia/core.py

```diff
@@ -911,5 +917,29 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def get_status_messages(self):
+        """Retrieve the messages to inform users about
+        the status of Gaia TAP
+        """
+        try:
+            subContext = self.GAIA_MESSAGES
+            connHandler = self._TapPlus__getconnhandler()
+            response = connHandler.execute_tapget(subContext, False)
+            if response.status == 200:
+                if isinstance(response, Iterable):
+                    for line in response:
+
+                        try:
+                            print(line.decode("utf-8").split('=', 1)[1])
+                        except ValueError as e:
+                            print(e)
+                            pass
+                        except IndexError as e:
+                            print("Archive down for maintenance")
+                            pass
```

Comment:
`pass` is a null operation which is meant for situations where the Python syntax requires an indented code block, but no code should be executed. There is no need to use `pass` here because the `print()` calls form the indented code blocks that Python expects. Furthermore, in the case of an `IndexError`, `e` would be an unused variable.
```suggestion
                        except ValueError as e:
                            print(e)
                        except IndexError:
                            print("Archive down for maintenance")
```